### PR TITLE
Split Elos

### PR DESCRIPTION
**Pull request by healeycodes**

### CODE REVIEW

File: stats.js [hunk 1]
```diff
@@ -84,14 +89,15 @@ function buildStats(data) {
...
-        player.elo += diff;
+        player.imposterElo += diff;
       } else {
-        diff = eloChange(player.elo, crewAvgElo)[1];
+        diff = eloChange(player.imposterElo, crewAvgElo)[1];
         player.imposterLoss += 1;
-        player.elo += diff;
+        player.imposterElo += diff;
       }
+      player.elo = (player.crewElo + player.imposterElo) / 2
```
Comment (al1y) [inline stats.js:100]: put a semicolon you fucking animal

File: stats.js [hunk 2]
```diff
@@ -55,25 +57,28 @@ function buildStats(data) {
...
-        player.elo += diff;
+        player.crewElo += diff;
       } else {
-        diff = eloChange(player.elo, imposterAvgElo)[1];
+        diff = eloChange(player.crewElo, imposterAvgElo)[1];
         player.crewLoss += 1;
-        player.elo += diff;
+        player.crewElo += diff;
       }
+      player.elo = (player.crewElo + player.imposterElo) / 2
```
Comment (al1y) [inline stats.js:81]:
```suggestion
      player.elo = (player.crewElo + player.imposterElo) / 2;
```

File: public/player.js [hunk 3]
```diff
@@ -10,7 +10,9 @@ fetch("/stats")
     const URLparts = document.location.href.split("/");
     const name = URLparts.pop() || URLparts.pop(); // handle potential trailing slash
     const player = json.players.filter(p => p.name === name)[0];
-    playerTitle.innerHTML += ` - ${name} (<code>${player.elo}</code>)`;
+    const elo = player.elo
```
Comment (al1y) [inline public/player.js]:
```suggestion
    const elo = player.elo;
```

File: stats.js [hunk 4]
```diff
@@ -55,25 +57,28 @@ function buildStats(data) {
...
 
     // Players are measured against the average of the other team
-    const avgElo = list =>
-      list.reduce((a, b) => a + players[b].elo, 0) / list.length;
-    let crewAvgElo = avgElo([...crew]);
-    let imposterAvgElo = avgElo(imposters);
+    const avgCrewElo = list =>
+      list.reduce((a, b) => a + players[b].crewElo, 0) / list.length;
+    const avgImposterElo = list =>
+      list.reduce((a, b) => a + players[b].imposterElo, 0) / list.length;
```
Comment (healeycodes) [inline stats.js]: Maybe a generic function here?

File: stats.js (hunk 1 above)
Comment (healeycodes) [reply stats.js:100]: 4/5 and 1/5 weighting perhaps? Due to 20% chance of being imposter in any game 😇

**Pull request merged**